src/com/dakl/Main.java

```diff
@@ -16,9 +16,12 @@ public static void main(String[] args) {
 
         CheckerBoard game = CheckerBoard.blankBoard();
 
+        // REVIEW : Setup method
         Space start_loc = game.getBoardSpace(Main.parseSpace("C4R5"));
         Team r = game.red;
+        // REVIEW : calling static methods
         Main.threeByThree(game.black, new Team[] {r, r, r, r}, start_loc, true, game);
+        // REVIEW : using public members
         game.getBoardSpace(("C5R2")).pc = game.red.roster.get(9);
```

Comment:
This is a high-level comment for a lot of the code : while Java code conventions around using public fields to access class members have been loosening up over the last couple of years, overall, the best practice is still to wrap access to class members with getters/setters. In general, public fields are generally OK if they are immutable.